wfsim/core.py

```diff
@@ -316,7 +318,7 @@ def photon_timings(self, t, n_photons, recoil_type):
             return
 
         if (self.config.get('s1_model_type') == 'simple' and 
-           recoil_type.lower() in ['er', 'nr']):
+           recoil_type in [1, 2]):
```

Comment:
Does this now conflict with the string input from epix (`er`/`nr`)? Should we add more types, according to the input we give to NEST?

Reply:
No it is only used for s1 scintillation models (and curently ignored even). Furthermore my epix strax plugin imports the dtype from wfsim